rpi/interrupts.go

```diff
@@ -202,4 +215,6 @@ func pigpioInterruptCallback(gpio, level int, rawTick uint32) {
 	default:
 		boardInstance.logger.Error("unknown digital interrupt type")
 	}
+	// store the current ticks for debouncing
+	interrupts.lastTicks = tick
```

Comment:
can multiple goroutines be running this function at the same time or does the boardInstance lock prevent that? If this variable doesn't get updated by the time the second subsequent tick gets processed then we wouldn't ignore it.

Reply:
the lock will prevent lastTicks from being modified/accessed by multiple triggers at the same time